### DISCUSSION TITLE
Interdependencies discussions (DEIC working groups)

### DISCUSSION THREAD

**Discussion by aiaa-deic** [aiaa-deic/computation-ai#3] [General]
Relative to dependencies between this working group and others in the Digital Engineering Integration Committee:
- Digital Twin
- Digital Thread
- DevSecOps
- Ecosystem
- Digital System Model

**Comment by arthurrge** [comment]
[Brief4compute+ai-2023SciTech-DEIC.pdf](https://github.com/aiaa-deic/computation-ai/files/11415067/Brief4compute%2Bai-2023SciTech-DEIC.pdf)

This is the slide / relevant accompanying slides from the discussion precipitating this effort.

**Reply by jdhaines** [reply]
Would you be interested in having this account be essentially an organization for other AIAA-DEIC repos? I'm leading the DevSecOps effort and I think we'd like to use GitHub as well. 

Or maybe we should create an org with a rep from each subcommittee with an admin account, then proper repos under it? 

I'd love your thoughts to grow together and correctly.

**Reply by arthurrge** [reply]
Very supportive of collaborating!
I'm not fluent in GitHub, however, so encourage any with seasoned perspective to help us get the structure right as early as possible. Please reach out!

**Reply by arthurrge** [reply]
Additional slides shown in today's call were just screen shots of this site - so please reference the above.

**Comment by arthurrge** [comment]
At today's kick-off we had:
-  Rick Graves (Ecosystems), 
- Melanie Lorang (Thread), 
- Jay Ganguili (Twin)

**Comment by arthurrge** [comment]
DECO Scope
- 
The scope of [DECO (Digital Ecosystem) ](https://github.com/aiaa-deic/deco/discussions) has matured as they move closer to their position paper and will aid in scoping out topics they will otherwise cover:
1. Datacenter vs. Cloud/elastic - including performance optimization and scalability
2. Architectural models & data models for interoperability and collaboration (APIs, standards, libraries, frameworks, IDEs, etc)
3. Governance of software, models, & data
4. Cross-system (ERP, PLM, MES, MRO, etc.) interoperability for digital thread

(Confirm / comments from @digitalhangarguy ?)

**Comment by arthurrge** [comment]
Other DEIC Scope
-
1. [DevSecOps](https://github.com/aiaa-deic/devsecops/discussions) coverage includes CI/CD and therefore some of the testing/VVUQ scope
2. The [Modeling Trust](https://github.com/orgs/aiaa-deic/discussions/categories/vvuq-framework-semantics) strategic initiative should cover the rest of the VVUQ aspects (model maturity assessment, predictive capability, continual calibration, etc.)

(Confirm/comments from @jdhaines & @opinonfischer ?)
